suites/nautilus/rgw/tier_1_rgw_multisite_primary_to_secondary.yaml

```diff
@@ -4,6 +4,22 @@
 #ceph-rgw1 is master/primary site
 #ceph-rgw2 is slave/secondary site
 
+# add "test-config" object to add custom io config for tests
+# example: 
+#  - test:
```

Comment:
Could we include other keys... I think this works only if we use `sanity_rgw.py`

Reply:
so this very particular to RGW only. this change is in multisite too.

Reply:
doc string added in `.py` files